src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -668,11 +668,26 @@ public static void Initialize(Collection<Exception> errors, List<string> moduleP
             else
             {
                 // DSC SxS scenario
-                var psAssembly = Assembly.GetEntryAssembly();
-                var configSystemPath = Path.GetDirectoryName(psAssembly.Location);
+                //var psAssembly = Assembly.GetEntryAssembly();
```

Comment:
The commented out code should be removed.

Reply:
Done.